chemie/shop/templates/shop/happy-hour.html

```diff
@@ -72,16 +71,8 @@
     })
       .then((willDelete) => {
```

Comment:
Will delete has the meaning "Start happy hour", right? If so, then name this input variable something like accept?